src/styles/card.scss

```diff
@@ -0,0 +1,39 @@
+@import 'variables';
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid $color-border-card;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    background-size: contain;
+    background-repeat: no-repeat;
+    margin: 32px 19px 36px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    width: 166px;
+    height: 36px;
+    margin: 0 16px 4px;
+    font-family: $font-roboto;
+    font-weight: $font-medium;
+    color: $color-font;
+  }
+
+  &__subtitle {
+    font-size: 10px;
+    line-height: 14px;
+    margin: 0 16px 16px;
+    font-family: $font-roboto;
+    font-weight: $font-regular;
+    height: 14px;
+    width: 98px;
+    color: $color-font-secondary;
+  }
+}
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to sort CSS properties in a specific order. For example, position-related properties should come first, followed by box model, typography, and visual styling. Please reorder the properties in `.card`, `.card__image`, `.card__title`, and `.card__subtitle` to adhere to this guideline.